login/LoginRequest.js

```diff
@@ -5,4 +5,4 @@ class LoginRequest{
     }
 };
 
-module.exports = LoginRequest;
\ No newline at end of file
+module.exports=LoginRequest;
```

Comment:
Having spaces before and after the assingment operator is actually correct, please follow @ptiforka 's suggestion and install eslint/add .editorconfig .
Here's a link on linting in node.js, you can search around yourself as well, https://medium.com/the-node-js-collection/why-and-how-to-use-eslint-in-your-project-742d0bc61ed7

Reply:
did it,thanks